addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -553,6 +553,56 @@ QUnit.test('chat window: open / close', async function (assert) {
     assert.verifySteps(['rpc:channel_fold/open']);
 });
 
+QUnit.test('chat window: focus on ESCAPE', async function (assert) {
+    assert.expect(1);
+
+    this.data['res.partner'].records = [{
+        email: "testpartnert@odoo.com",
+        id: 11,
+        name: "TestPartner",
+    }];
+    Object.assign(this.data.initMessaging, {
+        channel_slots: {
+            channel_channel: [
+                {
+                    channel_type: 'channel',
+                    id: 20,
+                    is_minimized: true,
+                    is_pinned: true,
+                    name: "General",
+                    state: 'open',
+                    fold_state: 'open',
+                    uuid: 'channel-20-uuid',
+                },
+                {
+                    channel_type: 'channel',
+                    id: 21,
+                    is_minimized: true,
+                    is_pinned: true,
+                    name: "other chat",
+                    state: 'open',
+                    fold_state: 'open',
+                    uuid: 'channel-21-uuid',
+                }
+            ],
+        },
+    });
+
+    await this.start({
+        hasChatWindow: true
+    });
```

Comment:
Between start and the first step, an assertion should be done about the initial state.
You should at least verify that 2 chat windows exist before trying to do anything with them.